ceph/README.md

```diff
@@ -14,13 +14,27 @@ To use in Rook, you need to write a manifest of the custom resource CephCluster
 Reference for updating image
 ----------------------------
 
-This image based on the Dockerfile produced by `ceph/ceph-container`.
-You can make the Dockerfile with following commands:
-```console
-git clone git@github.com:ceph/ceph-container.git
-cd ceph-container
-make FLAVORS=luminous,debian,9 stage
-# Then the Dockerfile will be located at ./staging/luminous-debian-9-x86_64/daemon-base
+This container uses a cybozu's own Ceph [cybozu/ceph][]:neco-release. It's because using dmcrypt devices that is already encrypted by users is not supported in the upstream's newest stable version (v15.2.1), but the function is  necessary for Neco.
```

Comment:
> a cybozu's own Ceph
It sounds like Cybozu has ownership of Ceph. I think customized, enhanced or patched Ceph is better in this context.